src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/components/ApplicationRequirements.vue

```diff
@@ -7,7 +7,7 @@
     </template>
     <template v-if="applicationStore.isDraftCertificateTypeOneYear">
       <ECEOneYearRequirements v-if="!applicationStore.isDraftApplicationRenewal" />
-      <ECEOneYearRenewalRequirements v-else />
+      <ECEOneYearRenewalRequirements v-else :expired="certificationStore.latestCertificateStatus == 'Expired'" />
```

Comment:
nitpik best to use === in javascript.

Reply:
I don't think we really need `===` in typescript but will add it in, doesn't hurt!